### PR TITLE
readme: fix windows build command

### PR DESCRIPTION
**Pull request by rerickson1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,14 +88,16 @@ west build -p -b mg100 -d ble_gateway_dm_firmware/build/mg100 ble_gateway_dm_fir
...
 -DCONFIG_LCZ_APPROTECT_STARTUP=y \
 -DCONFIG_LCZ_APPROTECT_STARTUP_INIT_LEVEL=3 \
 -DCONFIG_LCZ_APPROTECT_STARTUP_INIT_PRIORITY=2 \
--DCONFIG_SB_SIGNING_KEY_FILE=\"%CD%/ble_gateway_dm_firmware/config/keys/debug_priv_a.pem\" \
--DCONFIG_SB_PUBLIC_KEY_FILES=\"%CD%/ble_gateway_dm_firmware/config/keys/debug_pub_b.pem,config/keys/debug_pub_c.pem\" \
+-DCONFIG_SB_SIGNING_KEY_FILE=\"config/keys/debug_priv_a.pem\" \
+-DCONFIG_SB_PUBLIC_KEY_FILES=\"config/keys/debug_pub_b.pem,config/keys/debug_pub_c.pem\" \
 -Dmcuboot_CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256=y \
--Dmcuboot_CONFIG_BOOT_SIGNATURE_KEY_FILE=\\\"%CD%/ble_gateway_dm_firmware/config/keys/debug_priv_c.pem\\\"
+-Dmcuboot_CONFIG_BOOT_SIGNATURE_KEY_FILE=\\\"config/keys/debug_priv_c.pem\\\"
```
Comment (ahedin25) [inline README.md]: \\\"../../ble_gateway_dm_firmware/config/keys/debug_priv_c.pem/\\\"

Comment (ahedin25) [reply README.md]: keep three slashes

**Review by ahedin25 — APPROVED** [review]

**Review by ahedin25 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -44,12 +40,11 @@ v0.14.2 of the Zephyr SDK is recommended for building the firmware.
...
-> **WARNING:** Before building the firmware, be sure to install all python dependencies
+**All of the following commands assume starting from from the directory where you ran `west update`**
 
-From the directory where you ran `west update`, issue the following command:
+> **WARNING:** Before building the firmware, be sure to install all python dependencies with the following commands:
 
 ```
-# Linux, macOS and Windows
 pip3 install --user -r zephyr/scripts/requirements.txt
 pip3 install --user -r modules/lib/laird_connect/attributes/generator/requirements.txt
```
Comment (ahedin25) [inline README.md:50]: pip3 install -r nrf/scripts/requirements.txt

**Comment by rerickson1** [comment]
@ahedin25 ping for re-review

**Review by ahedin25 — APPROVED** [review]

**Review by ahedin25 — APPROVED** [review]

**Pull request merged**